libs/payments/ui/src/lib/client/components/Header/index.tsx

```diff
@@ -0,0 +1,372 @@
+      'https://relay.firefox.com/',
+      'mozilla-websites',
+      'moz-subplat',
+      'bento',
+      'relay',
+      'permanent'
+    ),
+    vpn: constructHrefWithUtm(
+      'https://vpn.mozilla.org/',
+      'mozilla-websites',
+      'moz-subplat',
+      'bento',
+      'vpn',
+      'permanent'
+    ),
+    pocket:
+      'https://app.adjust.com/hr2n0yz?redirect_macos=https%3A%2F%2Fgetpocket.com%2Fpocket-and-firefox&redirect_windows=https%3A%2F%2Fgetpocket.com%2Fpocket-and-firefox&engagement_type=fallback_click&fallback=https%3A%2F%2Fgetpocket.com%2Ffirefox_learnmore%3Fsrc%3Dff_bento&fallback_lp=https%3A%2F%2Fapps.apple.com%2Fapp%2Fpocket-save-read-grow%2Fid309601447',
+  };
+
+  const iconClassNames = 'inline-block w-5 -mb-1 me-1';
+
+  return (
+    <header
+      className="bg-white fixed flex justify-between items-center shadow h-16 left-0 top-0 mx-auto my-0 px-4 py-0 w-full z-40 tablet:h-20"
+      role="banner"
+      data-testid="header"
+    >
+      <div className="flex items-center">
+        <Image
+          src={mozillaIcon}
+          alt="Mozilla logo"
+          className="object-contain"
+          data-testid="branding"
+          width={120}
+        />
+      </div>
+      {localized && (
+        <div className="flex items-center leading-normal">
+          <Localized id="next-header-help">
+            <LinkExternal
+              href="https://support.mozilla.org/products/mozilla-account"
+              title="Help"
+              className="inline-block relative p-2 -m-2 z-[1] rounded hover:bg-grey-100"
+            >
+              <Image
+                src={helpIcon}
+                aria-label="Help"
+                alt="Help"
+                role="img"
+                className="w-5 text-violet-900"
+              />
+            </LinkExternal>
+          </Localized>
+          {/** Bento Menu */}
+          <div
+            className="relative self-center flex mx-2"
+            ref={bentoMenuInsideRef}
+          >
+            <Localized id="next-bento-menu">
+              <button
+                onClick={toggleBentoMenuRevealed}
+                title="Mozilla products"
+                aria-label="Mozilla products"
+                aria-expanded={!!isBentoMenuRevealed}
+                aria-haspopup="menu"
+                className="rounded p-2 mx-2 border-transparent hover:bg-grey-100 transition-standard desktop:mx-8"
+              >
+                <Image
+                  src={bentoIcon}
+                  alt="Mozilla Logo"
+                  className="w-5 text-violet-900"
+                />
+              </button>
+            </Localized>
+
+            {/** Bento Dropdown */}
+            {isBentoMenuRevealed && (
+              <div
+                className="w-full h-full fixed top-0 start-0 bg-white z-10
+                      mobileLandscape:h-auto mobileLandscape:drop-down-menu mobileLandscape:top-10 mobileLandscape:-start-52 desktop:-start-50"
+              >
```

Comment:
Note, this does not need to be addressed in this PR.

Ideally the Bento Dropdown could use something from the Radix Primitives library, looks like they do have a [dropdown menu](https://www.radix-ui.com/primitives/docs/components/dropdown-menu).

This would simplify the header implementation, and hooks `useClickOutsideEffect` and `useEscKeydownEffect` would probably not be necessary.

Reply:
This feels like maybe we should make an epic to wrap a number of such instances? I haven't done a dive into which components can be replaced by Radix Primitives, but I suspect its more than just this.

Reply:
Do any other instances come to mind where Radix Primitives should have been used?

It's a goal of SP3 that where possible and when it makes sense, that Radix Primitives are used. So I'd like to make sure that happens as much as possible.

That said, I'd prefer we prioritize landing working code and then having follow up PRs to switch in Radix, so I'm happy to have a follow on ticket.

Reply:
Not that I'm aware of, seems my suspicions are unfounded :)

Reply:
Created FXA-11035, and added an inline comment referencing it